jupyterhub-k8s-hub.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: jupyterhub-k8s-hub
+  version: 3.2.1
+  epoch: 0
+  description: Zero to JupyterHub with Kubernetes
+  copyright:
+    - license: BSD-3-Clause
+  dependencies:
+    runtime:
+      - python3
+      - py3-kubernetes-asyncio
+      - py3-jupyterhub
+      - py3-jupyterhub-firstuseauthenticator
+      - py3-jupyterhub-hmacauthenticator
+      - py3-jupyterhub-ldapauthenticator
+      - py3-jupyterhub-ltiauthenticator
+      - py3-jupyterhub-nativeauthenticator
+      - py3-jupyterhub-tmpauthenticator
+      - py3-nullauthenticator
+      - py3-oauthenticator
+      - py3-mwoauth
+      - py3-pyjwt
+      - py3-jupyterhub-kubespawner
+      - py3-pymysql
+      - py3-psycopg2
+      - py3-pycurl
+      - py3-sqlalchemy-cockroachdb
+      - py3-statsd
+      - py3-jupyterhub-idle-culler
+      - py3-tornado
+      - py3-dateutil
+      - configurable-http-proxy
+      - iptables
+      - tini
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/jupyterhub/zero-to-jupyterhub-k8s
+      tag: ${{package.version}}
+      expected-commit: c0a696f56433be64ac85a3bc6e0bff329285e80a
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/jupyterhub
+      cp -r jupyterhub/files/hub/* "${{targets.destdir}}"/etc/jupyterhub/
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binaries in the location expected by jupyterhub-k8s-hub"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/usr/local/etc/jupyterhub
+          ln -sf /etc/jupyterhub/jupyterhub_config.py ${{targets.subpkgdir}}/usr/local/etc/jupyterhub/jupyterhub_config.py
+          ln -sf /etc/jupyterhub/z2jh.py ${{targets.subpkgdir}}/usr/local/etc/jupyterhub/z2jh.py
+    dependencies:
+      provides:
+        - jupyterhub-k8s-hub
+
+update:
+  enabled: true
+  github:
+    identifier: jupyterhub/zero-to-jupyterhub-k8s
+    use-tag: true
```

Comment:
Looks like there are releases?
- https://github.com/jupyterhub/zero-to-jupyterhub-k8s/releases

```suggestion
```

Reply:
Done! I didn't notice the "Releases" on the GitHub UI.